fabric/base/docker-compose-base.yaml

```diff
@@ -11,6 +11,7 @@ services:
     container_name: orderer0.luther.systems
     image: $IMAGE_NS/fabric-orderer:$IMAGE_TAG
     environment:
+      - DOCKER_PROJECT_DIR
```

Comment:
I don't think this needs to be added to the environment blocks since it's not used by the app itself, right?  Maybe only the CLI container needs it?

Reply:
Correct, removed.